server/routes/quizzes.js

```diff
@@ -24,13 +24,57 @@ router.route('/:id').get((req, res) => {
 router.route('/:id').post((req, res) => {
```

Comment:
since this is in the quizzes route (not a 'decks' route) we should have the param name be :deckId to make it clear what the param references.  Otherwise the initial inference is that :id would refer to the quiz id